examples/wordCount/application-DAG.json

```diff
@@ -0,0 +1,63 @@
+{
+  "applicationName": "WordCount",
+  "nodes": [
+    {
+      "task": "1",
+      "name": "text2lines",
+      "type": "producer",
+      "description": "Splits text into lines.",
+      "executor": "container",
+      "container": {
+        "image": "text2lines-image",
+        "resources": {
+          "cpu": "1",
+          "memory": "1GB",
+          "task-complexity": "no_of_instructions"
+        }
+      },
+      "comm-task": {
+        "payload": ""
+      },
+      "downstream": ["2"]
+    },
+    {
+      "task": "2",
+      "name": "lines2words",
+      "type": "prosumer",
+      "description": "Splits lines into words (stream 1).",
+      "executor": "container",
+      "container": {
+        "image": "lines2words-image",
+        "resources": {
+          "cpu": "1",
+          "memory": "1GB",
+          "task-complexity": "no_of_instructions"
+        }
+      },
+      "comm-task": {
+        "payload": ""
+      },
```

Comment:
What is the purpose of this comm-task field?

Reply:
it's for the communication task-related field! this DAG can be used for co-scheduling computation and communication tasks later.

Reply:
Hmm. Not sure if I got it 😅 I guess we can keep it, but the structure and purpose of the individual fields should probably be clarified and described in more by a schema (e.g., using CDDL) afterward.

Reply:
<img width="1175" alt="Screenshot 2024-07-04 at 15 19 32" src="https://github.com/hsel-netsys/iceflow/assets/49976934/1fac7ab3-f572-488f-9540-fc41be958c33">
How big is the communication task and when to transfer the payload? the "comm-task" field can be used to describe the flow size.

Reply:
Hm. So how would you actually describe that then? By providing a `payloadSize` subfield? And how would we specify _when_ to transfer it? By defining a maximum publish interval? Just asking because there is only an empty string in the `payload` field at the moment.